src/routes/Manual.svelte

```diff
@@ -368,14 +455,20 @@ function handleMouseUp(event) { handleTouchEnd(event); }
                           class="activatable"
                           fill="none"
                           stroke={hexagon.color}
-                          strokeWidth="5px"
+                          stroke-width="5px"
                           strokeLinejoin="miter"
```

Comment:
The casing is different for React so this line is probably ignored as well.

Reply:
Changing stroke-width does change the width of the hexagons' outline.